src/get_measurements.F90

```diff
@@ -199,16 +203,17 @@ subroutine Get_Measurements(Ctrl, SAD_Chan, SPixel, MSI_Data, status)
                                  (dR_dT0(1) / dR_dTm(1))) ** 2
             end if
          else
-            if (len_trim(Ctrl%LUTClass) == 3) then
+            if (.not. Ctrl%use_new_meas_error) then
                ! Old LUTs approach (fixed uncertainty)
                ! Note: NEDR is squared when old LUT is read in.
                SPixel%Sy(i,i) = SAD_Chan(ii)%Solar%NEDR
             else
                ! New LUTs approach (varying uncertainty)
                ! Convert reflectance to radiance
                j = SPixel%ViewIdx(i)
-               Lx = (SPixel%Ym(i) * cos(SPixel%Geom%SolZen(j) * d2r) * &
-                     SAD_Chan(ii)%Solar%F0) / Pi
+               !Lx = (SPixel%Ym(i) * cos(SPixel%Geom%SolZen(j) * d2r) * &
+               !      SAD_Chan(ii)%Solar%F0) / Pi
+               Lx = (100.0 * SPixel%Ym(i) * SAD_Chan(ii)%Solar%F0) / Pi
```

Comment:
Add a comment explaining why we're removing the \cos\theta. We won't remember next time we go to war with The Angles.